java/query/src/test/java/sleeper/query/executor/QueryExecutorTest.java

```diff
@@ -80,30 +71,21 @@
 import static sleeper.configuration.properties.table.TableProperty.DATA_BUCKET;
 import static sleeper.configuration.properties.table.TableProperty.ITERATOR_CLASS_NAME;
 import static sleeper.configuration.properties.table.TableProperty.ITERATOR_CONFIG;
-import static sleeper.dynamodb.tools.GenericContainerAwsV1ClientHelper.buildAwsV1Client;
+import static sleeper.statestore.inmemory.StateStoreTestHelper.inMemoryStateStoreWithPartitions;
 
-@Testcontainers
-public class QueryExecutorIT {
-    protected static final int DYNAMO_PORT = 8000;
-    protected static AmazonDynamoDB dynamoDBClient;
+public class QueryExecutorTest {
     protected static ExecutorService executorService;
 
-    @Container
-    public static GenericContainer dynamoDb = new GenericContainer(CommonTestConstants.DYNAMODB_LOCAL_CONTAINER)
-            .withExposedPorts(DYNAMO_PORT);
-
     @TempDir
     public Path folder;
```

Comment:
The IO with the file system makes this test probably a bit too slow to be run with the unit tests, although it's much faster now, thanks! This class should probably still be QueryExecutorIT.